applications/main/infrared/resources/infrared/assets/tv.ir

```diff
@@ -1,1945 +1,2129 @@
-command: F4 00 00 00
-#
-name: Vol_dn
-type: parsed
-protocol: RCA
-address: 0F 00 00 00
-command: 74 00 00 00
-#
-name: Ch_next
-type: parsed
-protocol: RCA
-address: 0F 00 00 00
-command: B4 00 00 00
-#
-name: Ch_prev
-type: parsed
-protocol: RCA
-address: 0F 00 00 00
-command: 34 00 00 00
-#
-# Model: JTC Genesis 5.5
-#
-name: Mute
-type: parsed
-protocol: NECext
-address: 01 72 00 00
-command: 5C A3 00 00
-#
-name: Power
-type: parsed
-protocol: NECext
-address: 01 72 00 00
-command: 1E E1 00 00
-#
-name: Vol_up
-type: parsed
-protocol: NECext
-address: 01 72 00 00
-command: 0A F5 00 00
-#
-name: Vol_dn
-type: parsed
-protocol: NECext
-address: 01 72 00 00
-command: 06 F9 00 00
-#
-name: Ch_next
-type: parsed
-protocol: NECext
-address: 01 72 00 00
-command: 48 B7 00 00
-#
-name: Ch_prev
-type: parsed
-protocol: NECext
-address: 01 72 00 00
-command: 44 BB 00 00
-#
-# Koro Box
-#
-name: Power
-type: parsed
-protocol: NEC
-address: 01 00 00 00
-command: 18 00 00 00
-#
-# Toshiba Amazon TV
-#
-name: Vol_up
-type: parsed
-protocol: NEC
-address: 01 00 00 00
-command: 0E 00 00 00
-#
-name: Vol_dn
-type: parsed
-protocol: NEC
-address: 01 00 00 00
-command: 0F 00 00 00
-#
-name: Mute
-type: parsed
-protocol: NEC
-address: 01 00 00 00
-command: 17 00 00 00
-#
-# Visio TV
-#
-name: Power
-type: parsed
-protocol: NEC
-address: 04 00 00 00
-command: 08 00 00 00
-#
-name: Vol_up
-type: parsed
-protocol: NEC
-address: 04 00 00 00
-command: 02 00 00 00
-#
-name: Vol_dn
-type: parsed
-protocol: NEC
-address: 04 00 00 00
-command: 03 00 00 00
-#
-name: Ch_next
-type: parsed
-protocol: NEC
-address: 04 00 00 00
-command: 00 00 00 00
-#
-name: Ch_prev
-type: parsed
-protocol: NEC
-address: 04 00 00 00
-command: 01 00 00 00
-#
-name: Mute
-type: parsed
-protocol: NEC
-address: 04 00 00 00
-command: 09 00 00 00
-#
-# Emerson TV
-#
-name: Power
-type: parsed
-protocol: NECext
-address: 84 E0 00 00
-command: 20 DF 00 00
-#
-name: Ch_next
-type: parsed
-protocol: NECext
-address: 84 E0 00 00
-command: 50 AF 00 00
-#
-name: Ch_prev
-type: parsed
-protocol: NECext
-address: 84 E0 00 00
-command: 51 AE 00 00
-#
-name: Vol_up
-type: parsed
-protocol: NECext
-address: 84 E0 00 00
-command: 60 9F 00 00
-#
-name: Vol_dn
-type: parsed
-protocol: NECext
-address: 84 E0 00 00
-command: 61 9E 00 00
-#
-name: Mute
-type: parsed
-protocol: NECext
-address: 84 E0 00 00
-command: 64 9B 00 00
-#
-# TCL 75S451
-#
-name: Power
-type: parsed
-protocol: NECext
-address: EA C7 00 00
-command: 17 E8 00 00
-#
-name: Mute
-type: parsed
-protocol: NECext
-address: EA C7 00 00
-command: 20 DF 00 00
-#
-name: Vol_up
-type: parsed
-protocol: NECext
-address: EA C7 00 00
-command: 0F F0 00 00
-#
-name: Vol_dn
-type: parsed
-protocol: NECext
-address: EA C7 00 00
-command: 10 EF 00 00
-#
-name: Ch_next
-type: parsed
-protocol: NECext
-address: EA C7 00 00
-command: 19 E6 00 00
-#
-name: Ch_prev
-type: parsed
-protocol: NECext
-address: EA C7 00 00
-command: 33 CC 00 00
-
```

Comment:
have no clue why it's done this

Reply:
Because of hidden symbols and line endings. We have special script to cleanup ir files.

Reply:
just added a new draft pr, will have to check through the cleanup script and add the functionality into my IR appender after work today